README.md

```diff
@@ -1,2 +1,31 @@
 # Capstone-1
 My first capstone project at Microverse
+
+## Built With
+HTML, CSS, and JavaScript.
+
+## Getting Started
+To setup, follow these simple steps.
+
+- Clone the [repo](https://github.com/DJ-MrJay/Capstone-1) or download the .zip file. 
+- Run with live server or simply open index.html with your browser.
+
+## Author
+- GitHub: [@githubhandle](https://github.com/DJ-MrJay)
+- Twitter: [@twitterhandle](https://twitter.com/jonah_wambua)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mr-jay/)
+
+## Contributors
+Contributions and feature requests are welcome!
+
+## Show your support
+Give ⭐️⭐️⭐️⭐️⭐️ if you like this project!
+
+## Github Page
+This project is [live](https://dj-mrjay.github.io/Capstone-1/) on Github Pages.
```

Comment:
- [ ] Please add a video describing this project, this video should not be more than 5 mins